homeworks/vitali.timolyanov_XX-NF-XX/homework_7/task_02_login.js

```diff
@@ -0,0 +1,67 @@
+const MAX_FAILED_LOGINS = 3;
+const LOCK_TIME = 10000;
+
+const fs = require('fs');
+
+const loginFile = './task_02_logins.json';
+
+class Login {
+    constructor() {
+        this.logins = JSON.parse(fs.readFileSync(loginFile).toString().trim());
```

Comment:
I would always wrap `JSON.parse()` with `try...catch`... uncaught exception may cause you a surprise in the most unpredictable moment :)

Reply:
Thank you for your review.
Actually I made async file loading, then I added a few content checks and added logic that prevented any actions before file was loaded. But, right after that moment when code was done and working I thought about KISS, TDD, simplicity, actual task and replaced all that logic with just single line of code.
Now I understand that I should ask questions if not sure about my decisions.

Reply:
Thank you.
I'll make it a rule to wrap such things with `try...catch`.